BrownianMotion/Continuity/KolmogorovChentsov.lean

```diff
@@ -89,14 +97,87 @@ lemma constL_lt_top (hc : c ≠ ∞) (hd_pos : 0 < d) (hp_pos : 0 < p) (hdq_lt :
     constL T c d p q β < ∞ := by
   sorry
 
-theorem finite_kolmogorov_chentsov (hT : HasBoundedInternalCoveringNumber (Set.univ : Set T) c d)
+theorem finite_kolmogorov_chentsov (h_diam : EMetric.diam (.univ : Set T) < ∞)
+    (hT : HasBoundedInternalCoveringNumber (Set.univ : Set T) c d)
```

Comment:
The bounded covering number assumptions implies that the diameter is not infinite, so you can remove `h_diam`: see the new lemma https://github.com/RemyDegenne/brownian-motion/blob/e1110a71a5b96b0e00580afcbf170493b565fa8e/BrownianMotion/Continuity/HasBoundedInternalCoveringNumber.lean#L41 .